docs/design/router.md

```diff
@@ -178,67 +157,50 @@ druid.router.avatica.balancer.type=consistentHash
 
 This is a non-default implementation that is provided for experimentation purposes. The consistent hasher has longer setup times on initialization and when the set of Brokers changes, but has a faster Broker assignment time than the rendezous hasher when tested with 5 Brokers. Benchmarks for both implementations have been provided in `ConsistentHasherBenchmark` and `RendezvousHasherBenchmark`. The consistent hasher also requires locking, while the rendezvous hasher does not.
 
-HTTP endpoints
---------------
-
-The Router process exposes several HTTP endpoints for interactions.
-
-### GET
-
-* `/status`
-
-Returns the Druid version, loaded extensions, memory used, total memory and other useful information about the process.
 
-* `/druid/v2/datasources`
+### Example Production Configuration
```

Comment:
could you please go for sentence casing for the headings so "Example production configuration"

Reply:
thanks, I adjusted `indexer.md` and `router.md` headings to use sentence casing, I kept the capitalization in `configuration/index.md` for consistency with the rest of the page